pkgs/development/libraries/grpc/default.nix

```
@@ -1,13 +1,12 @@
-{ stdenv, fetchFromGitHub, cmake, zlib, c-ares, pkgconfig, openssl, protobuf, gflags }:
+{ stdenv, fetchgit, cmake, zlib, c-ares, pkgconfig, openssl, protobuf, gflags }:
 
 stdenv.mkDerivation rec {
-  version = "1.23.0"; # N.B: if you change this, change pythonPackages.grpcio and pythonPackages.grpcio-tools to a matching version too
+  version = "1.24.2"; # N.B: if you change this, change pythonPackages.grpcio and pythonPackages.grpcio-tools to a matching version too
   pname = "grpc";
-  src = fetchFromGitHub {
-    owner = "grpc";
-    repo = "grpc";
+  src = fetchgit {
```

please use fetchFromGitHub, as this will download a minimal tar.gz as opposed to checking out the entire git history
```suggestion
  src = fetchFromGitHub {
```